uber/models.py

```diff
@@ -642,6 +642,11 @@ def merch(self):
         merch = self.donation_swag
         if self.gets_shirt and DONATION_TIERS[SHIRT_LEVEL] not in merch:
             merch.append(DONATION_TIERS[SHIRT_LEVEL])
+        elif self.gets_free_shirt:
+            shirt = '2nd ' + DONATION_TIERS[SHIRT_LEVEL]
+            if self.takes_shifts and self.worked_hours < 6:
```

Comment:
This change was actually unrelated to the merge, but it was a fix that needed to be made.